app/DoctrineMigrations/Version20180502134816.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180502134816 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE `service` CHANGE COLUMN `guid` `guid` CHAR(36) CHARACTER SET \'utf8\' NULL COMMENT \'(DC2Type:guid)\' ;
```

Comment:
Is hier geen data migratie nodig? Values welke impliciet naar bv iets als een lege string zijn gezet nu expliciet naar NULL moeten worden omgezet?

Reply:
Deze is denk ik minder belangrijk dan de opmerking hieronder. Of de `guid` waarde op `''` of `null` staat maakt denk ik in de praktijk niet zoveel uit. Of heb je een goede rede om dit aan te merken?